tests/unit/info.tcl

```diff
@@ -334,21 +334,25 @@ start_server {tags {"info" "external:skip" "debug_defrag:skip"}} {
 
         test {stats: instantaneous metrics} {
             r config resetstat
+            r config set hz 100
             set retries 0
             for {set retries 1} {$retries < 4} {incr retries} {
-                after 1600 ;# hz is 10, wait for 16 cron tick so that sample array is fulfilled
+                after 1600 ;# Wait for 16 cron tick so that sample array is fulfilled
```

Comment:
With hz 100, it's 160 cron ticks?

```suggestion
                after 1600 ;# Wait for 160 cron tick so that sample array is fulfilled
```

Reply:
Sorry, I accidentally edited your comment on my phone instead of replying to it. Yeah, I actually tested a little bit more and it wasn't actually getting filled, so waiting a bit longer.